### PR TITLE
docs: Initial version of stakeholder requirements

### LINKED ISSUE #24: Initial version of stakeholder requirements
Provide initial version of stakeholder requirements as a separate Contribution Request

### PR DESCRIPTION
**Pull request by ramceb**
Issue-ref: closes #24

### CODE REVIEW

**Comment by qor-lb** [comment]
@FScholPer could you please coordinate a review with Fabian Frie and Alexander Warg? I currently don't see them as members in this project. This is the initial version for the stakeholder requirements for v0.5 and was already discussed in the Architecture Community.

File: docs/requirements/stakeholder/index.rst [hunk 1]
```diff
@@ -17,6 +17,739 @@
...
+
+.. stkh_req:: Enable cooperation via standardized APIs
+    :id: STKH_REQ__30
+    :reqtype: Non-Functional
+    :security: NO
+    :safety: QM
+    :rationale: To enable cooperation with other cooperation partners.
+    :status: valid
+
+    The software platform shall be based on open and/or standardized APIs.
```
Comment (FScholPer) [inline docs/requirements/stakeholder/index.rst:44]: what does that mean in context of open source?

File: docs/requirements/stakeholder/index.rst [hunk 2]
```diff
@@ -17,6 +17,739 @@
 Stakeholder Requirements
```
Comment (FScholPer) [inline docs/requirements/stakeholder/index.rst:17]: Is Stakeholder still the right wording. I think we had that point already. Another thing would be to have a description not only a headline would be great

File: docs/requirements/stakeholder/index.rst [hunk 3]
```diff
@@ -17,6 +17,739 @@
...
+
+.. stkh_req:: Variant management
+    :id: STKH_REQ__60
+    :reqtype: Functional
+    :security: NO
+    :safety: QM
+    :rationale: tbd
+    :status: valid
+
+    The software platform shall provide variant management support.
```
Comment (FScholPer) [inline docs/requirements/stakeholder/index.rst:54]: makes no sense in context of open source

File: docs/requirements/stakeholder/index.rst [hunk 4]
```diff
@@ -17,6 +17,739 @@
...
+
+.. stkh_req:: IP protection
+   :id: STKH_REQ__50
+   :reqtype: Non-Functional
+   :security: NO
+   :safety: QM
+   :rationale: tbd
+   :status: valid
+
+   The software platform shall support cooperation models, where partners do not
```
Comment (FScholPer) [inline docs/requirements/stakeholder/index.rst:68]: same here what does that mean?

File: docs/requirements/stakeholder/index.rst [hunk 5]
```diff
@@ -17,6 +17,739 @@
...
+   :status: valid
+
+   The software platform shall support the following automotive network
+   protocols
+
+   * SOME/IP
+   * DDS
+   * UWB including Driver for UWB Peripheral
+   * SPI (+ CSC ADI & Texas Instruments chipset support)
+   * Vehicle to Grid + ISO Charge protocols
```
Comment (FScholPer) [inline docs/requirements/stakeholder/index.rst:507]: Is that really in scope?

**Review by FScholPer — CHANGES_REQUESTED** [review]

File: docs/requirements/stakeholder/index.rst (hunk 2 above)
Comment (ramceb) [reply docs/requirements/stakeholder/index.rst:17]: I missed this discussion but would be open to change the wording here. Any suggestions?

File: docs/requirements/stakeholder/index.rst (hunk 3 above)
Comment (ramceb) [reply docs/requirements/stakeholder/index.rst:54]: Here I do not agree. The solution will be used in many different variants or configurations be it different hardware, different bus systems, different application sets etc.  Somehow this has to be supported.

File: docs/requirements/stakeholder/index.rst (hunk 4 above)
Comment (ramceb) [reply docs/requirements/stakeholder/index.rst:68]: There might be projects where the sw platform is hosting closed source applications.

File: docs/requirements/stakeholder/index.rst (hunk 5 above)
Comment (ramceb) [reply docs/requirements/stakeholder/index.rst:507]: On the long-term. Why not?

File: docs/requirements/stakeholder/index.rst (hunk 3 above)
Comment (FScholPer) [reply docs/requirements/stakeholder/index.rst:54]: Still I do not understand what a stakeholder is. We have project leads, committers and collaborators in Eclipse Projects. There is no other role.

File: docs/requirements/stakeholder/index.rst (hunk 1 above)
Comment (ramceb) [inline docs/requirements/stakeholder/index.rst:44]:
```suggestion
    The software platform shall where possible be based on existing standards (e.g. network protocols).
```

**Review by FScholPer — APPROVED** [review]
